package.json

```diff
@@ -26,18 +27,18 @@
   "author": "Shannon Moeller <me@shannonmoeller.com> (http://shannonmoeller.com)",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/shannonmoeller/svgstore/issues"
+    "url": "https://github.com/svgstore/svgstore/issues"
   },
-  "homepage": "https://github.com/shannonmoeller/svgstore",
+  "homepage": "https://github.com/svgstore/svgstore",
   "dependencies": {
-    "cheerio": "^0.20.0",
+    "cheerio": "^0.22.0",
     "object-assign": "^4.1.0"
   },
   "devDependencies": {
-    "ava": "0.15.2",
+    "ava": "0.16.0",
```

Comment:
My Greenkeeper dashboard has access to our `svgstore` projects, but when I try to enable the service, I get a "no admin permissions" warning. It's totally up to you if you want to change permissions or not, but I think GK can only be enabled by an Admin-level project collaborator.